Packs/AzureRiskyUsers/Integrations/AzureRiskyUsers/AzureRiskyUsers.py

```diff
@@ -0,0 +1,395 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+
+import requests
+from urllib.parse import urlparse
+from urllib.parse import parse_qs
+from typing import Dict, List, Optional, Any
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+class Client:
+    """
+    API Client to communicate with AzureRiskyUsers.
+    """
+
+    def __init__(self, client_id: str, verify: bool, proxy: bool):
+        self.ms_client = MicrosoftClient(
+            self_deployed=True,
+            auth_id=client_id,
+            token_retrieval_url='https://login.microsoftonline.com/organizations/oauth2/v2.0/token',
+            grant_type=DEVICE_CODE,
+            base_url='https://graph.microsoft.com/v1.0',
+            verify=verify,
+            proxy=proxy,
+            scope='https://graph.microsoft.com/IdentityRiskyUser.Read.All '
+                  'IdentityRiskEvent.ReadWrite.All IdentityRiskyUser.Read.All '
+                  'IdentityRiskyUser.ReadWrite.All offline_access')
+
+    def risky_users_list(self, risk_state: Optional[str], risk_level: Optional[str], limit: int,
```

Comment:
Let's make sure request functions keep this naming standard

```suggestion
    def risky_users_list_request(self, risk_state: Optional[str], risk_level: Optional[str], limit: int,
```

Reply:
Done